src/main.py

```diff
@@ -3,70 +3,33 @@
 """
 
 import os
-import streamlit as st
-import pandas as pd
-import numpy as np
-import plotly.express as px
-import plotly.graph_objects as go
-
 from datetime import datetime, timedelta
 
+import pandas as pd
+import streamlit as st
 from nowcasting_datamodel.connection import DatabaseConnection
 from nowcasting_datamodel.models.metric import MetricValue
-from get_data import get_metric_value
+
 from auth import check_password
-from status import status_page
 from forecast import forecast_page
+from get_data import get_metric_value
+from plots.all_gsps import make_all_gsps_plots
+from plots.forecast_horizon import (
+    make_mae_by_forecast_horizon,
+    make_mae_forecast_horizon_group_by_forecast_horizon,
+    make_mae_vs_froecast_horizon_group_by_date,
```

Comment:
hi @peterdudfield, this looks a lot tidier. I'm getting an error when I run the code locally, and I think it's because the function in `line 20` in `main.py` still has a typo.